src/index.d.ts

```diff
@@ -982,6 +987,123 @@ declare namespace admin.remoteConfig {
    */
   type RemoteConfigParameterValue = ExplicitParameterValue | InAppDefaultValue;
 
+  /**
+   * Interface representing a Remote Config template version.
+   * Output only, except for the version description. Contains metadata about a particular
+   * version of the Remote Config template. All fields are set at the time the specified Remote
+   * Config template was published. A version's description field may be specified in
+   * `publishTemplate` calls.
+   */
+  export interface Version {
+    /**
+     * The version number of a Remote Config template.
+     */
+    versionNumber?: string;
+
+    /**
+     * The timestamp of when this version of the Remote Config template was written to the
+     * Remote Config server.
```

Comment:
Suggest "backend" instead of server, unless it's inaccurate.  We use backend a lot in the guide docs.

Reply:
Thanks. `backend` makes more sense here.